model/DataObject.php

```diff
@@ -1744,44 +1745,79 @@ public function getManyManyComponents($componentName, $filter = "", $sort = "",
 	}
 
 	/**
+	 * @deprecated 4.0 Method has been renamed to hasOne()
+	 * @param string $component
+	 * @return array|null
+	 */
+	public function has_one($component = null) {
+		Deprecation::notice('4.0', 'Method has been renamed to hasOne()');
```

Comment:
Can our deprecation notices be a bit more verbose? eg: "belongs_to has been deprecated, please use belongsTo instead"